### PR TITLE
[LogicApps Integration Account] : Examples included for integration account Api's

### PR DESCRIPTION
**Pull request by pankajsn**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

**Comment by olydis** [comment]
@amar any clue what the schema validator is complaining about? (https://travis-ci.org/Azure/azure-rest-api-specs/jobs/208444863) it says "is an invalid JSON" for every example although it looks pretty valid to me - or is it a schema error?

**Comment by amarzavery** [comment]
@olydis - I am pulling the changes and trying to repro locally on my box. Will get back to you shortly.

File: arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+					   "qualifier":"ZZ",
+					   "value":"ZZ"
+					},
+					"receiverBusinessIdentity":{
+					   "qualifier":"AA",
+					   "value":"AA"
+					}
+				 }
+			  }
+		   },
```
Comment (amarzavery) [inline arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json]: The comma should be removed after closing curly brace

File: arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json [hunk 2]
```diff
@@ -0,0 +1,485 @@
+{
+   "parameters":{
+      "api-version":"2016-06-01",
+      "resourceGroupName":"<resourceGroupName>",
```
Comment (amarzavery) [inline arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json]: please provide sample values everywhere. "<resourceGroupName>" does not match the regex of resourceGroupName `[ '^[-\\w\\._\\(\\)]+$'`

**Comment by amarzavery** [comment]
After making some quick fixes for resourceGroupName in the example, I ran the model validator locally and found the following issue:

```
Azure swagger model validation using x-ms-examples and examples in spec
error: Found errors in validating the response with statusCode "200" for x-ms-example "Get IntegrationAccount callback URL" in operation "Integration
Accounts_GetCallbackUrl".:

{ code: 'RESPONSE_VALIDATION_ERROR',
  message: 'Found errors in validating the response with statusCode "200" for x-ms-example "Get IntegrationAccount callback URL" in operation "Integr
ationAccounts_GetCallbackUrl".',
  innerErrors:
   [ { code: 'INVALID_RESPONSE_BODY',
       errors:
        [ { code: 'OBJECT_ADDITIONAL_PROPERTIES',
            params: [ [ 'queries', 'basePath' ] ],
            message: 'Additional properties not allowed: queries,basePath',
            path: [],
            description: 'The callback url.' } ],
       message: 'Invalid body: Additional properties not allowed: queries,basePath',
       path: [] } ] }
{ AssertionError: swagger "/Users/amarz/sdk/azure-rest-api-specs/arm-logic/2016-06-01/swagger/logic.json" contains model validation errors.
    at /Users/amarz/sdk/azure-rest-api-specs/test/model.js:21:14
  name: 'AssertionError',
  actual: false,
  expected: true,
  operator: '==',
  message: 'swagger "/Users/amarz/sdk/azure-rest-api-specs/arm-logic/2016-06-01/swagger/logic.json" contains model validation errors.',
  generatedMessage: false }
    1) /Users/amarz/sdk/azure-rest-api-specs/arm-logic/2016-06-01/swagger/logic.json should have valid examples.
```

File: arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json (hunk 2 above)
Comment (pankajsn) [inline arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json]: @amarzavery 
fixed in all the examples

File: arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json (hunk 1 above)
Comment (pankajsn) [inline arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json]: @amarzavery  fixed

File: arm-logic/2016-06-01/examples/MapsCreateorUpdate.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (amarzavery) [inline arm-logic/2016-06-01/examples/MapsCreateorUpdate.json]: arm-logic/2016-06-01/examples/MapsCreateorUpdate.json -> MapsCreate **O** rUpdate.json; `UpperCase O`.

On linux and mac filenames are case sensitive.

File: arm-logic/2016-06-01/swagger/logic.json [hunk 4]
```diff
@@ -1767,6 +1806,9 @@
         ],
         "operationId": "Maps_CreateOrUpdate",
         "description": "Creates or updates an integration account map.",
+		"x-ms-examples": {
+          "Create or update a map": { "$ref": "../examples/MapsCreateOrUpdate.json" }
```
Comment (amarzavery) [inline arm-logic/2016-06-01/swagger/logic.json]: ../examples/MapsCreateOrUpdate.json" This should match the filename correctly.

File: arm-logic/2016-06-01/swagger/logic.json [hunk 5]
```diff
@@ -1767,6 +1806,9 @@
         ],
         "operationId": "Maps_CreateOrUpdate",
         "description": "Creates or updates an integration account map.",
+		"x-ms-examples": {
```
Comment (amarzavery) [inline arm-logic/2016-06-01/swagger/logic.json]: make sure the indentation is 2 spaces and not tabs. If you are using VSCode, then replace tabs with 2 spaces and press Alt + Shift  + F for auto formatting.

Comment (pankajsn) [inline arm-logic/2016-06-01/swagger/logic.json]: @amarzavery  fixed

File: arm-logic/2016-06-01/swagger/logic.json (hunk 4 above)
Comment (pankajsn) [inline arm-logic/2016-06-01/swagger/logic.json]: @amarzavery  fixed

File: arm-logic/2016-06-01/examples/MapsCreateorUpdate.json (hunk 3 above)
Comment (pankajsn) [inline arm-logic/2016-06-01/examples/MapsCreateorUpdate.json]: @amarzavery  fixed

**Review by olydis — APPROVED** [review]

**Pull request merged**